libs/blocks/global-navigation/utilities/getUserEntitlements.js

```diff
@@ -63,27 +136,33 @@ const getSubscriptions = async ({ queryParams, locale }) => {
  * @param {object} object required params
  * @param {array} object.params array of name value query parameters [{name: 'Q', value: 'PARAM'}]
  * @param {object} object.locale {country: 'CH', language: 'de'}
+ * @param {string} object.format format function, raw or default
  * @returns {object} JIL Entitlements
  */
-const getUserEntitlements = async ({ params, locale } = {}) => {
-  if (!window.adobeIMS?.isSignedInUser()) return Promise.reject(new Error('User not signed in'));
+const getUserEntitlements = async ({ params, locale, format } = {}) => {
+  if (!window.adobeIMS?.isSignedInUser()) return Promise.resolve(emptyEntitlements());
 
   const queryParams = getQueryParameters(params);
-  if (entitlements[queryParams]) return entitlements[queryParams];
+  if (entitlements[queryParams]) {
+    return format === 'raw'
+      ? entitlements[queryParams]
+      : entitlements[queryParams]
+        .then((res) => mapSubscriptionCodes(res));
```

Comment:
Would we also need a `catch` statement here?

Reply:
I'll try to revisit this comment when I write some tests in the future so I can actually check this exact scenario.
Added it for now